packages/cli/src/commands/env/index.ts

```diff
@@ -42,6 +42,13 @@ const help = () => {
 
   ${chalk.dim('Examples:')}
 
+  ${chalk.gray(
+    '–'
+  )} Pull all Development Environment Variables down from the cloud
+
+      ${chalk.cyan(`$ ${getPkgName()} env env pull`)}
+      ${chalk.cyan(`$ ${getPkgName()} env env pull .env.development.local`)}
```

Comment:
This looks like a bug: `env env`?

Reply:
I'm also no sure we want `vc env pull` to be used the same way as `vc pull`. Is there a reason to document this here?

Reply:
I noticed it was not mentioned here, but it was in the docs: https://vercel.com/docs/cli#commands/env

With the changes made to remove `vc pull --env some-file`, are we good here, or do you still think there's some unnecessary documentation?